src/components/pages/home/events.tsx

```diff
@@ -1,44 +1,124 @@
 'use client';
-import React, { useEffect, useRef, useState } from 'react';
+
+import { useState } from 'react';
 import events from '../../../data/events.json';
-import { MapPin } from '@phosphor-icons/react';
+import { MapPin } from 'phosphor-react';
 import EmptyEventCard from '../../no-events-card';
 import Image from 'next/image';
 import AddToCalendar from '@/components/AddToCalendar';
 
-type Event = {
-  communityName: string;
-  communityLogo: string;
-  eventName: string;
-  eventDate: string;
-  eventVenue: string;
-  eventLink: string;
-  location: string;
-};
-
 type EventCardProps = {
   communityName: string;
   title: string;
   date: string;
+  startTime: string;
```

Comment:
just the eventTime from the events.json should be more than enough @VarshiniSuresh90 

lets keep the details as what we have already requested from users.